### ISSUE TITLE
Hint Notification 

### ISSUE THREAD

**Issue by vanch3d** [SAFeSEA/openEssayist-slim#4]
When a ‘hint’ is opened, the focus is not taken to it until the very end of the page. This means it can be obscuring content until the user tabs through to dismiss it.
Focus should be passed immediately to the hint (i.e. its close button). Medium priority.

When there are two hints visible, and the user tabs to the first close button, the focus returns to the top of the page so the user has to tab through the one is then closed 
Focus should be placed on subsequent hints when other hints are closed.
